meson.build

```diff
@@ -7,7 +7,6 @@ project(
     'c_std=gnu17',     # Adds "-std=gnu17".  Includes GNU 17 extensions.
     'warning_level=2', # Adds "-Wextra".  Enables additional warnings. 
     'debug=true',      # Adds "-g".  Object files include debugging symbols.
-    'werror=true'      # Adds "-Werror".  Treat warnings as errors.
```

Comment:
We should continue treating warnings as errors.

Reply:
Good catch. Didn't intend to remove it.